exporter/stats/stackdriver/stackdriver.go

```diff
@@ -101,9 +101,25 @@ type Options struct {
 	Resource *monitoredrespb.MonitoredResource
 }
 
+// Enforces the singleton on NewExporter per process
+// lest there will be races with Stackdriver.
+// See Issue https://github.com/census-instrumentation/opencensus-go/issues/258
```

Comment:
You don't have to give a link to the issue here. The commit message for this change is going to mention the issue, we don't have to put links in the source code.

Reply:
Cool, I'll remove that.